src/systems/physics/Physics.cc

```diff
@@ -3259,7 +3259,11 @@ void PhysicsPrivate::UpdateCollisions(EntityComponentManager &_ecm)
         {
           msgs::Contact *contactMsg = contactsComp.add_contact();
           contactMsg->mutable_collision1()->set_id(_collEntity1);
+          contactMsg->mutable_collision1()->set_name(
+	    scopedName(_collEntity1, _ecm, ":", 1));
```

Comment:
I think the convention for scope delimiter is `::` in Gazebo.

Reply:
Done